src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -121,6 +122,13 @@ public class SynthTabbedPaneUI extends BasicTabbedPaneUI
 
     private boolean selectedTabIsPressed = false;
 
+    // Background color for selected tab and content pane
+    private Color selectColor;
+    // Background color for unselected tabs
+    private Color unselectedBackground;
```

Comment:
Can u recheck `unselectedBackground` purpose once again?

Reply:
`unselectedBackground` color is used to render the unselected tabs. What else you want me to check?

Reply:
`unselectedBackground` was not defined in GTKLookAndFeel class. Although the default graphics color for unselected tab was white but added the required property to make code consistent across LAFs.